deploy/kubernetes/deploy-driver.sh

```diff
@@ -15,6 +15,30 @@ set -o errexit
 
 readonly PKGDIR="${GOPATH}/src/sigs.k8s.io/gcp-compute-persistent-disk-csi-driver"
 readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
+#readonly BIND_ROLES="roles/compute.storageAdmin roles/iam.serviceAccountUser projects/${PROJECT}/roles/gcp_compute_persistent_disk_csi_driver_custom_role"
```

Comment:
why did this get moved? can it be moved back?

Reply:
We need to first extract the project sadly